src/main/webapp/WEB-INF/view/conversations.jsp

```diff
@@ -81,7 +84,7 @@
     <%
       for(Conversation conversation : conversations){
     %>
-      <li><a href="/chat/<%= conversation.getTitle() %>">
+      <a href="/chat/<%= conversation.getTitle() %>">
         <%= conversation.getTitle() %></a></li>
```

Comment:
It looks like you deleted the starting `<li>` tag but kept the closing `</li>` tag. Is this what you meant to do?

Reply:
No, and I think that was what I was looking for to get rid of an extra bullet so thank you, I fixed it.